docs/design/specs/Ecma-335-Augments.md

```diff
@@ -325,3 +325,19 @@ We propose the limit on #String and #Blob heap size is 2^29 (0.5 GB), that is an
 2) 0.5 GB is a very large heap. Having such a big PE file seems unreasonable and very rare scenario (if it exists at all).
 
 3) Having 3 spare bits available is very beneficial for the implementation. It allows to represent WinRT projected strings, namespaces, etc. in very efficient way. If we had to represent heap indices with all 32 bits it would bloat various structures and increase memory pressure. PE files over 0.5 GB of size are very rare, but the overhead would affect all compilers and tools working with the metadata reader.
+
+## Module Initializer
+
+All modules may have a module initializer. This method shall be static, a member of the module, take no parameters, return no value, be marked with **rtspecialname** and **specialname**, and be named `.cctor`.
```

Comment:
```suggestion
All modules may have a module initializer. This method shall be a global method, static, take no parameters, return no value, be marked with **rtspecialname** and **specialname**, and be named `.cctor`.
```

The ECMA spec does not explain "member of the module" anywhere. It does explain global methods.

Reply:
It does however refer to the special `<Module>` class. Describing the module cctor as being a member of the `<Module>` class would also be correct. However, I prefer Jan's formulation, as global methods are a concept in the standard, which is easier to understand. Also, global methods are defined as always being static, so it is redundant, although acceptable to describe the .cctor as a static global method.